app/views/notes/index.html.erb

```diff
@@ -39,13 +39,5 @@
   <%= render :partial => "notes/notes" %>
 
   <hr />
-  
 </div>
 
-<% if params[:controller] == "tag" || params[:controller] == "search" %>
-  <%= render :partial => "sidebar/related" %>
-<% elsif params[:action] == "author" || params[:action] == "author_topic" %>
-  <%= render :partial => "sidebar/author" %>
-<% else %>
-  <%= render :partial => "sidebar/user" %>
-<% end %>
```

Comment:
Hey @RuthNjeri the removal of this section is the one causing the tests to fail. Also I  don't think  we want to remove this sidebar...it gives access to recent wiki edits ...You might also want to add back the `set_sidebar` under  index in the controller. what do you think?

Reply:
Thanks @cesswairimu 😄 

The removal of that section was part of this [discussion](https://github.com/publiclab/plots2/pull/8800#issuecomment-740052184)  with @jywarren where we replace it with the sort dropdown for the notes...

I think a user will still be able to access the recent wiki edits via the wiki section that will be linked on the notes page, I am working on this [here](https://github.com/publiclab/plots2/pull/8848), therefore, access to the recent edits won't be completely lost, just changed to another page....

I think this can also be open to further discussion before the PR is merged..

Reply:
Oh okay, then I think we might need to rewrite that test or remove it